arangod/Agency/Inception.cpp

```diff
@@ -298,16 +298,19 @@ bool Inception::restartingActiveAgent() {
                 }
               }
               
+              auto const  theirConfigVP = comres->result->getBodyVelocyPack();
+              auto const& lcc           =
+                theirConfigVP->slice().get("configuration");
+              
               auto agency = std::make_shared<Builder>();
-              agency->openObject();
-              agency->add("term", theirConfig.get("term"));
-              agency->add("id", VPackValue(theirLeaderId));
-              agency->add("active",      tcc.get("active"));
-              agency->add("pool",        tcc.get("pool"));
-              agency->add("min ping",    tcc.get("min ping"));
-              agency->add("max ping",    tcc.get("max ping"));
-              agency->add("timeoutMult", tcc.get("timeoutMult"));
-              agency->close();
+              { VPackObjectBuilder b(agency.get()); 
+                agency->add("term", theirConfig.get("term"));
```

Comment:
should this pull from theirConfig, which might not be leader, or from theirConfigVP->slice()?  mild race condition pulling data from two different servers.

Reply:
ok. so the term is not crucial. it will part of RAFT. but will do that regardless. good point